src/List/_ListFooter.js

```diff
@@ -13,9 +13,9 @@ const ListFooter = ({
     );
 
     return (
-        <li {...props} className={ListItemClasses}>
+        <span {...props} className={ListItemClasses}>
```

Comment:
This seems right to change the footer to a span, but do we need to stay in line with fundamental-styles here?

I've looked at a few o their examples and they all still show the footer as a `li`, like here: https://github.com/SAP/fundamental-styles/blob/master/stories/list/standard/standard-list.stories.js#L301

Reply:
I guess this is part of 
> deviates from fundamental-styles markup for accessibility reasons

from your description, but is that OK to do?  Do we document our deviations?

Reply:
Trying to get this change in fundamental-styles: https://github.com/SAP/fundamental-styles/pull/1397

We can either jump the gun and get this change in fd-react or wait for the fd-styles change and subsequent fd-styles upgrade in fd-react.

Reply:
I'm for jumping the gun, as we will eventually be up to spec when fd-styles is upgraded :)